homeassistant/components/tplink/README.md

```diff
@@ -32,3 +32,8 @@ When doing so, do not forget to run `script/setup` to generate the translations.
 Other information like the category and whether to enable per default are read from the feature,
 as are information about units and display precision hints.
 
+In order to make the tests pass, you will also need to perform the following steps:
```

Comment:
Honestly, this doesn't belong here at all.

These are mostly generic instruction on how to use the HA devs environment, besides a fixture adjustment.

Reply:
Agreed, I simplified it a bit to include only the fixture edit + snapshot update, I hope that's fine? The idea is to make the update process as straightforward as possible for future developers.

Reply:
I removed all README.md updates from this PR, so this is resolved now.